crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -323,6 +325,7 @@ impl Action {
         let kind = match self {
             Action::Sequence(act) => Value::SequenceAction(act.to_raw()),
             Action::Transfer(act) => Value::TransferAction(act.to_raw()),
+            Action::ValidatorUpdate(act) => Value::ValidatorUpdateAction(act.clone().into()),
```

Comment:
I believe this clone can be avoided, it looks like `tendermint::validator::Update` should impl `Copy` because it's fields are both Copy, but they forgot to add that.

Reply:
hm, should i upstream that to tendermint-rs then?